src/components/code/code.js

```diff
@@ -1,20 +1,10 @@
-export const Code = ({ children }) => {
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { xonokai } from 'react-syntax-highlighter/dist/esm/styles/prism';
+export const Code = ({ children, language }) => {
```

Comment:
update this line `export const Code = ({ children, language = 'javascript' }) => {`